nf_core/download.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+""" Download a nf-core pipeline """
+
+from __future__ import print_function
+
+from io import BytesIO
+import logging
+import os
+import requests
+import subprocess
+import sys
+from zipfile import ZipFile
+
+
+import nf_core.list, nf_core.lint
+
+def download_workflow(pipeline, release=None, singularity=False, outdir=None):
```

Comment:
Just curious, Why make it as independent function rather than not including it in the Class below ?

Reply:
Can't remember, was copying my earlier code structure for the other modules.